keyboards/asmodus/plaquead/rgb/rgb.c

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2024 Asmodus
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "quantum.h"
+
+#ifdef RGB_MATRIX_ENABLE
+led_config_t g_led_config = { {
```

Comment:
This can and should be moved into the info.json.  `qmk info` can help with that.

Reply:
literally i don't know how to do with some led not in matrix 😥

Reply:
in the switch matrix?  Basically, you don't give them a matrix position.   A good example is the corne keyboard, as it has 6 underglow LEDs on each half, and then rest are key LEDs.  

But the qmk info command should help render that, IIRC.

Reply:
will cook that tonight

Reply:
cooked